examples/redis/example-redis.py

```diff
@@ -2,10 +2,9 @@
 from rgsync.Connectors import RedisConnector, RedisConnection, RedisClusterConnection
 
 '''
-Create Redis Cluster Connection
+Create Redis Connection
 '''
-cnodes = [{"host": "127.0.0.1", "port": "7001"}, {"host": "127.0.0.1", "port": "7002"}, {"host": "127.0.0.1", "port": "7003"}]
-r_conn = RedisClusterConnection(host=None, port=None, cluster_nodes=cnodes)
+r_conn = RedisClusterConnection(host='127.0.0.1', port=7001)
```

Comment:
I guess here its RedisConnection?

Reply:
copy-paste mistake I will correct it.

Reply:
we didn't fix that, did we?

Reply:
right, @ratnesh-tiwari  lets fix and merge

Reply:
In latest commit I had removed this file and added two separate files for example of standalone and cluster.
examples/redis/example-redis-cluster.py
examples/redis/example-redis-standalone.py

Reply:
@ratnesh-tiwari so why do we still see this file here?

Reply:
@MeirShpilraien 
Yes there is still mistake in examples I am going to resolve it.